LAD-AMA-Common/metrics_ext_utils/metrics_ext_handler.py

```diff
@@ -433,7 +434,7 @@ def start_metrics(is_lad):
 
         metrics_pid_path = me_config_dir + "metrics_pid.txt"
 
-        binary_exec_command = "{0} -TokenSource MSI -Input influxdb_udp -InfluxDbHost 127.0.0.1 -InfluxDbUdpPort {1} -DataDirectory {2} -LocalControlChannel -MonitoringAccount {3} -LogLevel Error".format(metrics_ext_bin, me_influx_port, me_config_dir, monitoringAccount)
+        binary_exec_command = "{0} -TokenSource AMCS -Input influxdb_udp,otlp_grpc,otlp_grpc_prom -InfluxDbHost 127.0.0.1 -InfluxDbUdpPort {1} -LogLevel Error".format(metrics_ext_bin, me_influx_port)
```

Comment:
can we use these new arguments only if is_lad is false? otherwise no need to change current parameters for LAD